packages/headless/src/extensions/drag-and-drop.tsx

```diff
@@ -12,6 +12,17 @@ export interface DragHandleOptions {
 }
 function absoluteRect(node: Element) {
   const data = node.getBoundingClientRect();
+  const modal = node.closest('[role="dialog"]');
```

Comment:
should we also add something about this to the docs, I don't know if all people use this attr for their dialog`[role="dialog"]`. Wyt?